code/modules/jobs/job_types/assistant.dm

```diff
@@ -33,7 +33,7 @@ Assistant
 		/obj/item/crowbar/large = 1
 	)
 
-	job_flags = JOB_ANNOUNCE_ARRIVAL | JOB_CREW_MANIFEST | JOB_EQUIP_RANK | JOB_CREW_MEMBER | JOB_NEW_PLAYER_JOINABLE | JOB_REOPEN_ON_ROUNDSTART_LOSS | JOB_ASSIGN_QUIRKS | JOB_CAN_BE_INTERN
+	job_flags = STATION_JOB_FLAGS | JOB_CANNOT_OPEN_SLOTS
```

Comment:
if you want to go the extra mile, we should instead give this flag to whatever the overflow job is, in case the station trait messes w/ it.

Reply:
sure